lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -395,6 +398,49 @@
       - Specifies the order of operations when rolling out an updated task.
       - Corresponds to the C(--update-order) option of C(docker service create).
       - Requires API version >= 1.29.
+  rollback_delay:
+    type: int
+    default: 0
+    description:
+      - Delay between task rollbacks.
+      - Corresponds to the C(--rollback-delay) option of C(docker service create).
+      - Requires API version >= 1.28.
+  rollback_parallelism:
+    type: int
+    default: 1
+    description:
+      - Maximum number of tasks rolled back simultaneously (0 to roll back all at once).
+      - Corresponds to the C(--rollback-parallelism) option of C(docker service create).
+      - Requires API version >= 1.28.
+  rollback_failure_action:
+    type: str
+    default: pause
+    description:
+      - Action to take in case of container failure.
+      - Corresponds to the C(--rollback-failure-action) option of C(docker service create).
+      - Requires API version >= 1.28.
+    choices:
+      - continue
+      - pause
+  rollback_monitor:
+    type: int
+    description:
+      - Duration after each task rollback to monitor for failure.
+      - Corresponds to the C(--rollback-monitor) option of C(docker service create).
+      - Requires API version >= 1.28.
+  rollback_max_failure_ratio:
+    type: float
+    description:
+      - Failure rate to tolerate during a rollback.
+      - Corresponds to the C(--rollback-max-failure-ratio) option of C(docker service create).
+      - Requires API version >= 1.28.
+  rollback_order:
+    type: str
+    default: stop-first
+    description:
+      - Rollback order ("start-first"|"stop-first").
```

Comment:
Should be added as `choices` instead. See `restart_policy` for an example. They also needs to be set as choices in the `argument_spec`.